src/core/addonManager.js

```diff
@@ -24,9 +21,16 @@ export function getAddon(guid, { _mozAddonManager = window.navigator.mozAddonMan
     });
 }
 
-export function install(_url, eventCallback,
-  { _mozAddonManager = window.navigator.mozAddonManager } = {}) {
-  const url = addQueryParams(_url, { src: 'discovery-promo' });
+export function install(_url, eventCallback, {
+  _mozAddonManager = window.navigator.mozAddonManager,
+    src,
```

Comment:
My indenter wants it to go there, not sure why. I moved all the args to the next line.